client/elements/menus/sc-suttaplex-share-menu.js

```diff
@@ -156,20 +156,43 @@ class SCSuttaplexShareMenu extends ReduxMixin(Localized(PolymerElement)) {
     }
   }
 
+  _createCopyableTextArea(inputText) {
+    const isRTL = document.documentElement.getAttribute('dir') === 'rtl';
+    const textArea = document.createElement('textarea');
+    // Prevent zooming on iOS
+    textArea.style.fontSize = '12pt';
+    // Reset box model
+    textArea.style.border = '0';
+    textArea.style.padding = '0';
+    textArea.style.margin = '0';
+    // Move element out of screen horizontally
+    textArea.style.position = 'absolute';
+    textArea.style[ isRTL ? 'right' : 'left' ] = '-9999px';
+    // Move element to the same position vertically
+    let yPosition = window.pageYOffset || document.documentElement.scrollTop;
+    textArea.style.top = `${yPosition}px`;
+
+    textArea.setAttribute('readonly', '');
+    textArea.value = inputText;
+
+    return textArea;
+  }
+
   // copies inputtext to the clipboard by creating and selecting a dummy element.
   _copyToClipboard(inputText) {
```

Comment:
could you extract this to a separate module? (take away _createCopyableTextArea with it)

Reply:
done